scrnaseq.Rmd

```diff
@@ -541,7 +891,12 @@ sc = Seurat::FindNeighbors(sc, dims=1:param$pc_n)
 # Cluster using the Leiden algorithm
 # Paper to Leiden algorithm: https://www.nature.com/articles/s41598-019-41695-z
 # Seurat vignette suggests resolution parameter between 0.4-1.2 for datasets of about 3k cells
-sc = Seurat::FindClusters(sc, resolution=param$cluster_resolution, algorithm=4)
+sc = Seurat::FindClusters(sc, resolution=param$cluster_resolution, algorithm=1)
```

Comment:
Why change algorithm=4 (Leiden - you recommended that a while back) to 1 (original Louvain)?

Reply:
Fixed.